app/collins/controllers/IpmiApi.scala

```diff
@@ -41,7 +41,7 @@ trait IpmiApi {
   val IPMI_FORM = Form(
     mapping(
       "username" -> optional(text(1)),
-      "password" -> optional(text(8)),
+      "password" -> optional(text(minLength=5, maxLength=16)),
```

Comment:
My only comment is that in IPMI 2.0 the max password length is 20. Maybe bump this to that? I believe earlier IPMI versions (where the max was 16 chars) will truncate the password if its too long, but I'm not 100%.